openhantek/src/openhantek.cpp

```diff
@@ -62,7 +62,11 @@ OpenHantekMainWindow::OpenHantekMainWindow(QWidget *parent,
 
   // Create the controller for the oscilloscope, provides channel count for
   // settings
-  this->dsoControl = new Hantek::Control();
+  timer = new QTimer();
```

Comment:
I know that this ancient code base doesn't do it, but please initialize new member variables in the header file (QTimer *timer = new QTimer();). This is possible since c++11. I'm working on integrating firmware loading, and I have done this for old members in many places as well.

Reply:
@davidgraeff  In-class init will not work unless you replace forward-declaration
```
class QTimer;
```
with 
```
#include <QTimer>
```
Is it the change you are applying to other places?